src/scripts/cf/form-tags/Tag.ts

```diff
@@ -210,17 +272,34 @@ namespace cf {
 			let isValid: boolean = true;
 			let valueText: string = value.text;
 
-			if(this.pattern){
-				isValid = this.pattern.test(valueText);
```

Comment:
@SampsonCrowley why you would have the custom validation attributes instead of the pattern attribute?